src/Fhir.Anonymizer.Shared.Core/AnonymizationConfigurations/AnonymizationFhirPathRule.cs

```diff
@@ -15,10 +16,13 @@ public class AnonymizationFhirPathRule : AnonymizerRule
 
         public string ResourceType { get; private set; }
 
+        public string ReplaceWith { get; set; }
```

Comment:
Same here, we should not add specific properties here.

Reply:
Thanks, fixed